local.properties

```diff
@@ -0,0 +1,10 @@
+## This file must *NOT* be checked into Version Control Systems,
+# as it contains information specific to your local configuration.
+#
+# Location of the SDK. This is only used by Gradle.
+# For customization when using a Version Control System, please read the
+# header note.
+#Wed Dec 27 00:54:49 IST 2023
+sdk.dir=C\:\\Users\\TAMOJIT\\AppData\\Local\\Android\\Sdk
+TEST_USER_NAME=Colt
+TEST_USER_PASSWORD=Tamo@1997
```

Comment:
Please `git remove` this file.

Reply:
Do you want me to remove it or restore it??

Reply:
Remove

Reply:
Removed